daemon/logger/loginfo.go

```diff
@@ -127,3 +129,45 @@ func (info *Info) ImageFullID() string {
 func (info *Info) ImageName() string {
 	return info.ContainerImageName
 }
+
+// InfoMap returns information about a container as a map.
+func (info *Info) InfoMap(keyMod func(string) string) map[string]string {
```

Comment:
@cpuguy83 Just to clarify, because I might have misunderstood you the first time: Would you like me to just get rid of the `keyMod` parameter, the `InfoMap` function or "put everything in fluentd-logger implementation"?

Preparing this for a more "generalized implementation" was originally suggested by @thaJeztah ...